includes/admin/class-wc-stripe-inbox-notes.php

```diff
@@ -17,13 +17,9 @@ class WC_Stripe_Inbox_Notes {
 	const CAMPAIGN_2020_CLEANUP_ACTION = 'wc_stripe_apple_pay_2020_cleanup';
 
 	public function __construct() {
-		if ( ! WC_Stripe_UPE_Compatibility::are_inbox_notes_supported() ) {
```

Comment:
this validation will always return false, since admin-note store is not yet loaded when this class is instantiated. I moved it in the action methods.

Reply:
So, I guess the UPE availability notice wasn't working before?

I'd suggest we move the function `are_inbox_notes_supported` from `WC_Stripe_UPE_Compatibility` to this class now since we're applying it to other notes that are unrelated to UPE/Stripe Link.

Reply:
> So, I guess the UPE availability notice wasn't working before?
> 
indeed.